database/migrations/2022_06_03_192302_add_subtitle_to_stop.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Stop;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('stop', function (Blueprint $table) {
+            Stop::all()->each(function ($stop) {
+                if (isset($stop->stop_content['subtitle'])) {
+                    return;
+                }
+
+                // add an empty {} LocalizedText object
+                $stop->stop_content = [
+                    ...$stop->stop_content,
+                    'subtitle' => NULL,
```

Comment:
I assume doing this with the spread operator is just for clarity (vs `$stop->stop_content['subtitle'] = null`)?

It's a minor nitpick, but the comment here could potentially be confusing to someone coming in with a fresh set of eyes, since we're not actually adding an empty localizedText object, we're adding a placeholder to allow the Vue app to instantiate a localizedtext object. It's obvious, but on the other hand, if you're coming from a language that does lots of Magic™ you might look at that comment and wonder whether there's more going on than on first glance?

Reply:
Helpful feedback, thank you! Yeah, my FP JS accent is pretty strong here. I'll make the update.